examples/server.rs

```diff
@@ -17,6 +17,8 @@ use smoltcp::socket::SocketSet;
 use smoltcp::socket::{UdpSocket, UdpSocketBuffer, UdpPacketMetadata};
 use smoltcp::socket::{TcpSocket, TcpSocketBuffer};
 use smoltcp::time::{Duration, Instant};
+#[cfg(feature = "fragmentation-ipv4")]
+use smoltcp::iface::{FragmentsSet, FragmentedPacket};
```

Comment:
Done. On a side note, fragmentation is currently a part of the default features. If I remove it, it makes building the server_fragmented example really verbose (you have to enumerate all features). Is there some better way how to do that?

Reply:
I think it's fine to have fragmentation in default features as it's required in IPv4. API consumers can always not initialize the FragmentSet (which will happen by default), so this would only impact code size, and anyone conscious about code size should already turn off unused features.